src/System.Collections.Specialized/src/System/Collections/Specialized/OrderedDictionary.cs

```diff
@@ -81,7 +81,11 @@ public int Count
         {
             get
```

Comment:
This is fine but as an aside this could of course be written as
```c#
get {
   return _objectsArray?.Count ?? 0;
}
```

Reply:
I personally don't like the compressed style but I ready fix this if you request.

Reply:
It's your choice

Reply:
Leave as is.